nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment/src/main/java/org/nuxeo/ecm/platform/comment/impl/TreeCommentManager.java

```diff
@@ -107,132 +111,143 @@
      * Counts how many comments where made by a specific user on a specific document.
      */
     protected static final String QUERY_GET_COMMENTS_UUID_BY_COMMENT_ANCESTOR_AND_AUTHOR = //
-            QUERY_GET_COMMENTS_UUID_BY_COMMENT_ANCESTOR + " AND " + COMMENT_AUTHOR + " = '%s'";
-
-    /** @since 11.1 **/
-    public static final String COMMENT_RELATED_TEXT_ID = "commentRelatedTextId_%s";
+            QUERY_GET_COMMENTS_UUID_BY_COMMENT_ANCESTOR + " AND " + COMMENT_AUTHOR_PROPERTY + " = '%s'";
 
     @Override
-    public List<DocumentModel> getComments(CoreSession s, DocumentModel docModel) {
+    public List<DocumentModel> getComments(CoreSession session, DocumentModel docModel) {
         // Check permissions
-        checkReadCommentPermissions(s, docModel.getRef());
-        return CoreInstance.doPrivileged(s, session -> {
-            PageProvider<DocumentModel> pageProvider = getCommentsPageProvider(session, docModel, null, null, true);
+        checkReadCommentPermissions(session, docModel.getRef());
+        return CoreInstance.doPrivileged(session, s -> {
```

Comment:
perhaps i am missing something, but if i remember when i did the work on comments, reviews that i had suggests to put `session` as a parameter in the lambda

Reply:
Indeed, @troger suggested to inverse the logic because we're used to have `session` as the user session. We did only the change on the non deprecated class.